extensions/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>com.tomtom.kotlin</groupId>
         <artifactId>kotlin-tools</artifactId>
-        <version>1.6.2</version>
+        <version>1.6.3</version>
```

Comment:
Should this be a 'z'-increment (usually a fix) or a 'y' one?

Reply:
We tend to use 'y' increments for new features (whole new modules are 1.2.z -> 1.3.x, etc). Compared to that, this change seems negligible so to me it feels like a z-increment. I don't mind changing it to y if you prefer, but then I think we should be treating new features/modules as x-increments.